ui/src/views/Tasks/TaskGroup/index.jsx

```diff
@@ -745,7 +844,8 @@ export default class TaskGroup extends Component {
                 tooltipOpen
                 key={action.title}
                 FabProps={{
-                  disabled: actionLoading,
+                  disabled:
+                    actionLoading || this.groupActionDisabled(action.name),
```

Comment:
What does this do?

Reply:
So there is a popup menu like on the task view page. And it will have few options. For example if task group is already sealed, corresponding option would be disabled, because it doesn't make sense to seal it twice.